src/lib/updateGrantsRecipients.js

```diff
@@ -432,6 +436,30 @@ export async function processFiles(hashSumHex) {
 
       await Promise.all(grantReplacementPromises);
 
+      // ---
+      // Update GroupGrants
+      const HOUR_AGO = new Date(new Date() - 60 * 60 * 1000);
+
+      const replacements = await GrantReplacements.findAll({
+        where: { updatedAt: { [Op.gte]: HOUR_AGO } },
+        attributes: ['replacedGrantId', 'replacingGrantId'],
+      });
+
+      const affectedGroupGrants = await GroupGrant.findAll({
+        where: { grantId: uniq(replacements.map((g) => g.replacedGrantId)) },
+        attributes: ['id', 'grantId', 'groupId'],
+      });
+
+      for (const g of affectedGroupGrants) {
+        if (replacements.some((r) => r.replacedGrantId === g.grantId)) {
+          await GroupGrant.destroy({ where: { id: g.id } });
+          for (const replacement of replacements.filter((r) => r.replacedGrantId === g.grantId)) {
+            await GroupGrant.create({ groupId: g.groupId, grantId: replacement.replacingGrantId });
+          }
+        }
+      }
```

Comment:
I think there is an issue here. if two grants become one grant, both old values will be destroyed, which is correct. But two copies of the replacing grant will try to be added. The create needs to be a separate deduped loop.